__mocks__/googleapis.js

```diff
@@ -0,0 +1,376 @@
+const fs = jest.genMockFromModule('googleapis');
+
+let moment = require('moment');
```

Comment:
As generic rule, we use `let` if and only if the variable will be modified, otherwise use `const`, it makes clear weather or not we are creating a variable or a constant

Reply:
Gone through all but google auth code and made changes - will make those other changes later after work or something as it's just tedious and takes time